README.md

```diff
@@ -1,6 +1,6 @@
-## Third Party Honeycomb Projects
+## Open Source Honeycomb Projects
```

Comment:
```suggestion
## Third Party Open Source Honeycomb Projects
```

This should still be differentiated from regular Open Source projects, of which we have many.

Reply:
@JamieDanielson This works for me!!!